src/main/java/xyz/thepathfinder/android/ClusterListener.java

```diff
@@ -10,56 +10,56 @@
  *
  * @author David Robinson
  */
-public interface ClusterListener extends Listener<Cluster> {
+public class ClusterListener extends Listener<Cluster> {
```

Comment:
This should be abstract. We don't want developers instantiating ClusterListeners.